data/json/items/book/cooking.json

```diff
@@ -442,6 +442,42 @@
     "time": "35 m",
     "fun": -1
   },
+  {
+    "id": "vinegar_maker",
+    "type": "BOOK",
+    "name": { "str": "The Vinegar Maker's Handbook", "str_pl": "copies of Vinegar Maker's Handbook" },
```

Comment:
You could make this a winemaker's book, and add must recipes to it to decrease that bloat as well. Vinegar would naturally be in there as it's effectively ruined wine.

Reply:
I think a wine book could be separate, there appear to be books devoted solely to vinegar making (they cover vinegar making and things you can do with vinegar) and winemaking. Then perhaps there could be another fermenting book that covers both?

Reply:
That's a good idea.  I based it off an actual vinegar recipe book, which is why I increased the components for what you can make vinegar from.   So I'll just combine that and fruit wines.  Also I put in a note that I think mycus_wine should be learned from a flag you get from a mycus_mutation.  If not a flag then I think mycus wine should stay autolearned at a high cooking level and not be in a pre-cataclysm book.